carapace-server/src/main/javacc/RequestMatchParser.jj

```diff
@@ -51,79 +52,88 @@ SKIP :
 TOKEN :
 {
     < ALL: "all" > |
-    < REGEXP_DEF: "regexp" > |
+    < HTTPS: "https" > |
     < AND: "and" > |
     < OR: "or" > |
     < NOT: "not" > |
-    < HTTPS: "https" > |
     < O_BRACKET: "(" > |
     < C_BRACKET: ")" > |
-    < REGEXP: "'" (~["'"])* "'" >
+    < EQUALS: "=" > |
+    < REGEXP_DEF: "~" > |
+    < STRING: "\"" (~["\""])* "\"" > { matchedToken.image = matchedToken.image.substring(1, matchedToken.image.length() - 1);} |
+    < NAME: (["a"-"z","A"-"Z","_"])(["a"-"z","A"-"Z","0"-"9","-","_","."])* >
 }
 
 /* Main method to call to build a proper RequestMatcher over given matchingCondition
     *   - all
     *   - https
-    *   - regexp '.*index.*'
-    *   - regexp '.*index.*' and not https
+    *   - request.uri ~ ".*index.*"
+    *   - request.uri ~ ".*index.*" and not https
```

Comment:
what about "secure" instead of "https" ?
when we will move to HTTP/2.0 (or 3) we won't have "https" anymore :-)